### DISCUSSION TITLE
New Option to --> Apply Patterns: Apply Pattern to the whole Note

### DISCUSSION THREAD

**Discussion by FelipeRearden** [jglev/obsidian-apply-patterns-plugin#4] [Ideas]
[More info about me here](https://github.com/publicus/obsidian-apply-patterns-plugin/discussions/3)

Hello @publicus!!! 

Would be amazing to have a third option to apply a pattern to the whole note. This way I don’t need to `select all` before use the Apply Pattern hotkey. 
![Screen Shot 2021-08-10 at 08 41 44](https://user-images.githubusercontent.com/85180696/128861103-cd24daa0-1dba-44b4-8908-1989d3ccb0af.png)

Let me know if you need more information.

Thanks and have a great day!

**Comment by jglev** [comment]
Thank you for writing, @FelipeRearden!

This makes good sense to me, too. I hadn't targeted it initially because the functionality is currently achievable by selecting the whole document, as you noted. I've added it in the draft of the next version (#11)!

**Reply by FelipeRearden** [reply]
Hello again @publicus 

Thank you so much for this new feature. I know it's a small change, but for repeated workflow it's a good option to avoid one more hotkey (`cmd+a`) to select all :)

Thank you for adding this feature :)

**Reply by FelipeRearden** [reply]
@publicus 

The new feature is running perfect in 1.2.0 on Obsidian 0.12.10 on MacOS

For me, this issue is **closed**

Thanks !!!
